### DISCUSSION TITLE
Need help sourcing a RAW SHIM

### DISCUSSION THREAD

**Discussion by FearedFusionX** [3kh0/ext-remover#661] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Can someone source a raw shim with trogdor on it?
I only need it for *educational* purposes.

**Comment by FearedFusionX** [comment]
SH1mmer says that I need to source my own Raw SHIM somehow because`trogdor` isn't on the list. I want to know how to obtain the trogdor Raw SHIM if possible or if someone can source it for me.

**Reply by 3kh0** [reply, marked as answer]
Look up your chromebook model on google and see if you can find any, if not, oh well
